apps/condo/domains/common/utils/next/organization.tsx

```diff
@@ -0,0 +1,112 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
+import {
+    GetActiveOrganizationEmployeeDocument,
+    GetActiveOrganizationEmployeeQuery,
+    GetActiveOrganizationEmployeeQueryVariables,
+    GetActiveOrganizationEmployeeQueryResult,
+} from '@app/condo/gql'
+import { getCookie, setCookie, deleteCookie } from 'cookies-next'
+import cookie from 'js-cookie'
+import get from 'lodash/get'
+import { NextPageContext } from 'next'
+
+import { isSSR } from '@open-condo/miniapp-utils'
+
+
+export const ACTIVE_EMPLOYEE_COOKIE_NAME = 'organizationLinkId'
+
+function getCookieEmployeeId (context: NextPageContext): string | null {
+    let state: string | null
+    if (isSSR()) {
+        state = getCookie(ACTIVE_EMPLOYEE_COOKIE_NAME, { req: context.req, res: context.res })
+    } else {
+        try {
+            state = cookie.get(ACTIVE_EMPLOYEE_COOKIE_NAME)
+        } catch (e) {
+            console.error('Failed to get employee id from cookie', e)
+            state = null
+        }
+    }
+    return state || null
+}
+
+function setCookieEmployeeId (context: NextPageContext, activeEmployeeId: string) {
+    if (isSSR()) {
+        setCookie(ACTIVE_EMPLOYEE_COOKIE_NAME, activeEmployeeId, { req: context.req, res: context.res })
```

Comment:
Please use only `cookies-next` for cookie. It can determine automatically if it's in server or client environment (by req / res options).

Second lib is unnecessary here